File: CS101/CS101-DZ08-AleksaCekic4173/CS101-DZ08-zadatak03/src/cs101/dz08/zadatak3/Proizvod.java
package cs101.dz08.zadatak3;

public class Proizvod {
    private String ime;
    private int cena;

    public Proizvod() {
    }

    public Proizvod(String ime, int cena) {
        this.ime = ime;
        this.cena = cena;
    }

    public String getIme() {
        return ime;
    }

    public void setIme(String ime) {
        this.ime = ime;
    }

    public int getCena() {
        return cena;
    }

    public void setCena(int cena) {
        this.cena = cena;
    }

    @Override
    public String toString() {
        return "Proizvod{" + "ime=" + ime + ", cena=" + cena + '}';
    }
    
    
}

File: CS323/CS323-DZ10/CS323-DZ10/Proizvod.cpp
#include "Proizvod.h"


Proizvod::Proizvod() {

}

Proizvod::Proizvod(string ime, string barKod, string sifra, double cena) {
	this->ime = ime;
	this->barKod = barKod;
	this->sifra = sifra;
	this->cena = cena;
}

Proizvod::~Proizvod() {
	cout << "Proizvod je obrisan" << endl;
}

void Proizvod::setIme(string ime) {
	this->ime = ime;
}

void Proizvod::setBarKod(string barKod) {
	this->barKod = barKod;
}

void Proizvod::setSifra(string sifra) {
	this->sifra = sifra;
}

void Proizvod::setCena(double cena) {
	this->cena = cena;
}

string Proizvod::getIme() const {
	return ime;
}
string Proizvod::getBarKod() const {
	return barKod;
}
string Proizvod::getSifra() const {
	return sifra;
}
double Proizvod::getCena() const {
	return cena;
}

void Proizvod::info() {
	cout << "Cena proizvoda je " << this->cena << endl;
}
